index.bs

```diff
@@ -301,19 +301,19 @@ The <dfn method for="StorageBucket">persist()</dfn> method steps are:
 
     1. Let |bucket| be [=this=]'s [=/storage bucket=].
 
-    1. If |bucket|'s [=storage bucket/removed=] flag is true, [=reject=] |p| with {{InvalidStateError}}.
+    1. If |bucket|'s [=storage bucket/removed=] flag is true, [=queue a task=] to [=reject=] |p| with {{InvalidStateError}}.
     
-    1. If |bucket|'s [=bucket mode=] is `"persistent"`, [=/resolve=] |p| with true.
+    1. If |bucket|'s [=bucket mode=] is `"persistent"`, [=queue a task=] to [=/resolve=] |p| with true.
 
     1. Otherwise,
 
         1. Let |environment| be [=/this=]'s [=/relevant settings object=].
 
         1. Let |permission| be the result of [=getting the current permission state=] with `"persistent-storage"` and |environment|.
 
-        1. If |permission| is "{{PermissionState/granted}}", then set |bucket|'s [=bucket mode=] to `"persistent"` and [=/resolve=] |p| with true.
+        1. If |permission| is "{{PermissionState/granted}}", then set |bucket|'s [=bucket mode=] to `"persistent"` and [=queue a task=] to [=/resolve=] |p| with true.
 
-        1. Otherwise, [=/resolve=] |p| with false.
+        1. Otherwise, [=queue a task=] to [=/resolve=] |p| with false.
```

Comment:
When there are two steps that both resolve the promise like this I could see moving them into a parallel steps section.  But this is probably fine too.  I don't want to make you go back and forth too much.